source-control/contributing/readme.md

```diff
@@ -87,45 +87,45 @@ Agree if you want a linear or non-linear commit history. There are pros and cons
 
 ### Approach for non-linear commit history
 
-Merging `topic` into `master`
+Merging `topic` into `main`
 
 ```md
   A---B---C topic
  /         \
-D---E---F---G---H master
+D---E---F---G---H main
 
 git fetch origin
-git checkout master
+git checkout main
 git merge topic
 ```
 
 ### Two approaches to achieve a linear commit history
 
-#### Rebase topic branch before merging into master
+#### Rebase topic branch before merging into main
 
-Before merging `topic` into `master`, we rebase `topic` with the   :
+Before merging `topic` into `main`, we rebase `topic` with the   :
 
 ```bash
           A---B---C topic
          /         \
-D---E---F-----------G---H master
+D---E---F-----------G---H main
 
 git fetch origin
-git rebase master topic
-git checkout master
+git rebase main topic
```

Comment:
Would it make sense to continue to use the git defaults in these examples? Could be confusing since the main branch for git is still usually `master`

Reply:
Just saw in the issue that its mentioned Github might be switching to main, but at least as of now, all the git-scm documentation still references master

Reply:
Agree.

Reply:
Actually, I think this is applicable regardless of the merge target branch name. Whether `master`, `develop`, or whatever. I'm going to leave this for now, and if someone gets confused by the doc, they can file an issue :).